### PR TITLE
finish smoderp preparations for a pip package

### PR DESCRIPTION
**Pull request by pesekon2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,24 +35,32 @@ git clone https://github.com/storm-fsv-cvut/smoderp2d.git
...
-docker build -t smoderp docker/
+docker build -t smoderp2d docker/
 ```
 
 Run SMODERP command line tool from Docker container:
 
 ```sh
 docker run -v `pwd`:/opt/smoderp2d -w /opt/smoderp2d/ --rm --entrypoint \
- ./bin/start-smoderp2d.py smoderp \
+ ./bin/start-smoderp2d.py smoderp2d \
```
Comment (landam) [inline README.md:45]: Docker image is currently broken:

```
Traceback (most recent call last):
  File "/opt/smoderp2d/./bin/start-smoderp2d.py", line 15, in <module>
    start_smoderp2d()
  File "/opt/smoderp2d/bin/start.py", line 6, in start_smoderp2d
    from smoderp2d.runners.base import Runner
ModuleNotFoundError: No module named 'smoderp2d'
```

File: bin/arcgis/SMODERP2D.pyt [hunk 2]
```diff
@@ -8,7 +8,6 @@ py3 = sys.version_info[0] == 3
 if py3:
     from importlib import reload
 
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
```
Comment (landam) [inline bin/arcgis/SMODERP2D.pyt:11]: I am not aware of any easy way to install smoderp2d Python package in ArcGIS Pro.  This change will complicate SMODERP2D usage on this platform. Must be discussed.

**Review by landam — CHANGES_REQUESTED** [review]

File: .github/workflows/cmd_template.yml [hunk 3]
```diff
@@ -33,6 +33,11 @@ jobs:
           pip install -r .github/workflows/requirements.txt
           pip install -r .github/workflows/requirements${{ matrix.python-version }}.txt
 
+      - name: Install smoderp2d
+        run: |
+          pip wheel .
+          pip install smoderp2d-*.whl
```
Comment (landam) [inline .github/workflows/cmd_template.yml:39]: Why not calling 

```
pip install .
```

instead of

```
pip wheel .
pip install smoderp2d-*.whl
```
?

Comment (pesekon2) [reply .github/workflows/cmd_template.yml]: It used to cause some problems at my side but the problems were apparently solved in some other commits as I don't face them anymore. Changed to yours in [761fc34](https://github.com/storm-fsv-cvut/smoderp2d/pull/325/commits/761fc34ce88a682ee9a5fe72e6f90416db7cbe2b).

File: README.md (hunk 1 above)
Comment (landam) [reply README.md:47]: Solved by f2c3e5e

File: bin/arcgis/SMODERP2D.pyt (hunk 2 above)
Comment (landam) [reply bin/arcgis/SMODERP2D.pyt:11]: Solved by [564629d](https://github.com/storm-fsv-cvut/smoderp2d/pull/325/commits/564629d9a0c2948401e3258cc137c1337ef067d6)

**Review by landam — APPROVED** [review]
ArcGIS toolbox modified
QGIS tested with PYTHONPATH
Readme modified

**Pull request merged**